Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -65,7 +73,25 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
         if let source = source {
             return source.current.count
         }
-        return super.tableView(tableView, numberOfRowsInSection: section)
+        return suggestedSites.count
```

Comment:
Nit: Can reduce these lines to  
`return source?.current.count ?? suggestedSites.count`